diff --git a/src/lib/efreet_desktop.c b/src/lib/efreet_desktop.c
index 244e13b..e4f8894 100644
--- a/src/lib/efreet_desktop.c
+++ b/src/lib/efreet_desktop.c
@@ -199,7 +199,7 @@ efreet_desktop_init(void)
 
         efreet_desktop_changes_listen();
 
-        ecore_exe_run(PACKAGE_LIB_DIR "/efreet/efreet_desktop_cache_create", NULL);
+        ecore_exe_run(PACKAGE_LIB_DIR "/efreet/ver-svn-06/efreet_desktop_cache_create", NULL);
 
     }
 
@@ -1495,7 +1495,7 @@ efreet_desktop_update_cache_job(void *data __UNUSED__)
     fl.l_type = F_WRLCK;
     fl.l_whence = SEEK_SET;
     if (fcntl(efreet_desktop_exe_lock, F_SETLK, &fl) < 0) goto error;
-    efreet_desktop_exe = ecore_exe_run(PACKAGE_LIB_DIR "/efreet/efreet_desktop_cache_create", NULL);
+    efreet_desktop_exe = ecore_exe_run(PACKAGE_LIB_DIR "/efreet/ver-svn-06/efreet_desktop_cache_create", NULL);
     if (!efreet_desktop_exe) goto error;
 
     return;
