diff --git a/configure.ac b/configure.ac
index c856fca..0a69a99 100644
--- a/configure.ac
+++ b/configure.ac
@@ -8,6 +8,7 @@ AC_CANONICAL_BUILD
 AC_CANONICAL_HOST
 
 AM_INIT_AUTOMAKE(1.6 dist-bzip2)
+AM_MAINTAINER_MODE
 AM_CONFIG_HEADER(config.h)
 
 AC_ISC_POSIX
diff --git a/src/lib/Makefile.am b/src/lib/Makefile.am
index 72ba741..e2674fb 100644
--- a/src/lib/Makefile.am
+++ b/src/lib/Makefile.am
@@ -63,7 +63,7 @@ installed_headersdir = $(prefix)/include/efreet
 installed_headers_DATA = $(EFREETHEADERS) $(EFREETMIMEHEADERS) $(EFREETTRASHHEADERS)
 
 libefreet_la_LIBADD = @ECORE_LIBS@ @EINA_LIBS@
-libefreet_la_LDFLAGS = -version-info @version_info@
+libefreet_la_LDFLAGS = -release @PACKAGE_VERSION@$(DEBIAN_MINOR)
 
 libefreet_mime_la_LIBADD = @ECORE_LIBS@ @EINA_LIBS@ libefreet.la
 libefreet_trash_la_LIBADD = @ECORE_LIBS@ @EINA_LIBS@ libefreet.la
